service/javadsl/integration-tests/src/test/java/com/lightbend/lagom/it/mocks/AdditionalRoutersServiceTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.it.mocks;
+
+import akka.actor.ActorSystem;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import com.lightbend.lagom.javadsl.testkit.ServiceTest;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import play.mvc.Http;
+import play.mvc.Result;
+import static com.lightbend.lagom.javadsl.testkit.ServiceTest.defaultSetup;
+import static com.lightbend.lagom.javadsl.testkit.ServiceTest.startServer;
+import static org.junit.Assert.assertEquals;
+import play.test.*;
+
+
+import static play.test.Helpers.*;
+
+public class AdditionalRoutersServiceTest {
+
+    private static ServiceTest.TestServer server;
+    private static Materializer materializer;
+
+    @BeforeClass
+    public static void setUp() {
+        server = startServer(defaultSetup()
+            .withCluster(false).withCassandra(false)
+            .configureBuilder(b -> b.bindings(new AdditionalRoutersServiceModule()))
+        );
+        materializer = server.materializer();
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        if (server != null) {
+            server.stop();
+            server = null;
+        }
+    }
+
+
+    /** call the ping router (instance + bind dsl prefix) */
+    @Test
+    public void testInstanceRouterWithBindingPrefix() throws Exception {
+        Http.RequestBuilder request = Helpers.fakeRequest(GET, "/ping/");
```

Comment:
@marcospereira, can you shed a light on this? 

The `Router` impl being used is the `SimpleRouter` and it does add an extra `/` when adding a prefix, as observed [here](https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/routing/Router.scala#L173). 

And that explains why I need to add an extra `/` (ie: `/ping/` instead of simply `/ping`), but is that the expect behaviour? 

Maybe the `SimpleRouter` is not that often used which may explain why this is only coming to the surface now.